website/orders/templates/orders/shift_admin_footer.html

```diff
@@ -41,6 +41,15 @@
                                     <p class="font-footer footer-button-text">Add orders</p>
                                 </div>
                             </div>
+                            <div class="col expand-mobile">
+                                <a target="_blank" :href="`{% url 'admin:orders_order_add' %}?shift=${shift.id}&prioritize=1`">
```

Comment:
What does the prioritize field do? Why do we have it?

Reply:
That's the tosticie priority (in GET to pre-fill the checkbox on the add view)

Reply:
Yeah I know, but why do we set it for that button. They just get the priority if they place the order. They don't need to place orders via the admin

Reply:
Because this button is there to make it easier for admins to bypass the max number of orders, which we used to do through the django shift admin